pymc_experimental/statespace/core/statespace.py

```diff
@@ -833,6 +851,54 @@ def build_statespace_graph(
         if return_updates:
             return updates
 
+    def build_as_prior(
+        self,
+        name: str,
+        n_steps: int,
+        register_statespace_matrices: Optional[bool] = False,
+        k_endog: Optional[int] = None,
+        mode: Optional[str] = None,
+        **kwargs,
+    ) -> LinearGaussianStateSpaceRV:
+        """
+        Construct a random variable over
```

Comment:
Unfinished sentence: Construct a random variable over...?

Reply:
Oh, I accidentally created a new branch from a branch that had some other work on it. I will delete this function entirely, it doesn't work.